src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+// import logo from './logo.svg';
+import './index.css';
+import Calculator from './Calculor/calculor';
 
 function App() {
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Use class-based components (All components should be class-based).` ", kindly convert your `App.js` component which is currently a functional component to a class-based component.